src/main/resources/db.migration/LIB_V2024.03.04.0000__create_tables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE library_app.author (
+    author_id VARCHAR(36) NOT NULL,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
```

Comment:
I think `first_name` and `last_name` should be NOT NULL :D It will be strange to have only uuid as an author

Reply:
Done